### PR TITLE
Issue/55

### PR DESCRIPTION
**Pull request by shahnawaz19415**

### CODE REVIEW

File: client/src/test/login.test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  beforeEach(() => {
+    wrapper = mount(<WrappedComponent />);
+  });
+
+  it('Component should render without crashing', async () => {
+    expect(wrapper.find('form').length).toEqual(1);
+  });
+
+  it('Component should trigger login', () => {
```
Comment (mashhoodr) [inline client/src/test/login.test.js]: @shahnawaz19415 what about testing the validations

Comment (shahnawaz19415) [reply client/src/test/login.test.js]: Redux form validations?

Comment (mashhoodr) [reply client/src/test/login.test.js]: yes

Comment (shahnawaz19415) [reply client/src/test/login.test.js]: will add a check.

File: client/src/store/configureStore.dev.js [hunk 2]
```diff
@@ -26,4 +26,4 @@ const configureStore = preloadedState => {
   return store;
 };
 
-export default configureStore;
+export default configureStore();
```
Comment (oahmedqureshi) [inline client/src/store/configureStore.dev.js]: Why exporting default is calling function instead to invoking. Variable of this module will not hide in closure may produce bugs.

File: client/src/test/login.test.js (hunk 1 above)
Comment (shahnawaz19415) [reply client/src/test/login.test.js]: Redux fields need to be tested in isolation. Separate tests are required to check if fields work properly or not.

**Pull request merged**